schematic/utils/validate_utils.py

```diff
@@ -76,10 +77,18 @@ def np_array_to_str_list(np_array):
 
 def iterable_to_str_list(iterable):
     "Parse an iterable into a list of strings"
```

Comment:
You might want to update the docstring to describe the inputs that are allowed (str, Number, iterable)

Reply:
good point, thanks!